operator/images/cluster-setup/bin/install.sh

```diff
@@ -108,6 +108,58 @@ switch_cluster() {
   fi
 }
 
+fetch_bitwarden_secrets() {
+  CREDENTIALS_DIR="$WORKSPACE_DIR/credentials"
+
+  BW_CLIENTID=${BW_CLIENTID:-}
+  BW_CLIENTSECRET=${BW_CLIENTSECRET:-}
+  BW_PASSWORD=${BW_PASSWORD:-}
+
+  if [ -z "$BW_PASSWORD" ]; then
+    printf "Unset environment variable: \$BW_PASSWORD" >&2
+    return
```

Comment:
Shouldn't that be `exit 1`?

Reply:
We need install.sh to proceed if BW_PASSWORD is not set. (there is the install_clusters func that would run after this)

Reply:
1. if BW_PASSWORD is empty we have nothing to print right ?
2. just for esthetic/readability you can remove the else block and use just
```if [ -z "$BW_PASSWORD" ]; then return ;fi

Reply:
1. Right. It might be confusing for an user to read a message about Bitwarden when they don't intend to use it. I have removed the print statement.
2. Done.

Reply:
As discussed, the presence of `credentials/bitwarden.yaml` will trigger a check for the `BW_*` variables.